charts/kcp-operator/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
+# This supports shell glob matching, relative path matching, and
+# negation (prefixed with !). Only one pattern per line.
+.DS_Store
+# Common VCS dirs
+.git/
+.gitignore
+.bzr/
+.bzrignore
+.hg/
+.hgignore
+.svn/
```

Comment:
Why are these listed here? I am personally not a big fan of these giant monster gitignore files that mention every single possible name for temporary files that was ever used since Grace Hopper still wrote code with a pencil. IMHO it's not up to every single project under the sun to keep and maintain an evergrowing list of temp name patterns until the heat death of the universe.